pyanaconda/payload/rpmostreepayload.py

```diff
@@ -392,11 +392,26 @@ def _prepare_mount_targets(self):
         # matching NSS configuration inside Anaconda, and we can't "chroot" to
         # get it because that would require mounting the API filesystems in the
         # target.
+        cmd = "systemd-tmpfiles"
         for varsubdir in ('home', 'roothome', 'lib/rpm', 'opt', 'srv',
                           'usrlocal', 'mnt', 'media', 'spool', 'spool/mail'):
-            self._safe_exec_with_redirect("systemd-tmpfiles",
-                                          ["--create", "--boot", "--root=" + conf.target.system_root,
-                                           "--prefix=/var/" + varsubdir])
+            argv = ["--create", "--boot", "--root=" + conf.target.system_root,
+                    "--prefix=/var/" + varsubdir]
+
+            rc = util.execWithRedirect(cmd, argv)
+
+            # According to systemd-tmpfiles(8), the return values are:
+            #  0 → success
+            # 65 → so some lines had to be ignored, but no other errors
+            # 73 → configuration ok, but could not be created
+            #  1 → other error
+            # Therefore we ignore error 65, since this is coming from
+            # the payload itself and the actual execution of it was fine
+            if rc not in [0, 65]:
+                exn = PayloadInstallError(
+                    "%s %s exited with code %d" % (cmd, argv, rc))
```

Comment:
I'm not 100% sure about the following, but maybe someone else can comment on that.

My own instinct would be to use here something along the lines `"{} failed to create /var/{}".format(cmd, varsubdir)` to make the message more human when it is displayed. Detailed debug output is available in the logs: All executions are logged with complete options, return code and even captured output in `program.log` and also `journal`.

Reply:
Sure, this was just taken from the function it is replacing in the intention to keep the behaviour similar, but I think it makes sense to change it to something more readable.